devel/to-f90/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem		1.0
+PortGroup		compilers 1.0
+
+name			to-f90
+version			2000-03-21
+revision		1
+categories		devel
+platforms		darwin
+license			Noncommercial
+maintainers		{@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage		http://cd.textfiles.com/simtel/simtel0101/simtel/fortran/
+master_sites	${homepage}
+distname		to_f90
+use_zip			yes
+extract.mkdir	yes
+description		Fortran 77 to Fortran 90 converter
+
+long_description \
+A Fortran 90 program for converting fixed-format Fortran 77 code to \
+Fortran 90 free-format.  Other changes replace old Fortran constructs \
+with more modern ones. \
+ \
+The conversions made include: \
+1. Standard format (column 7-72) converted to free-format. \
+2. DO loops and IF blocks indented. \
+3. Most labels are removed from loops and replaced with END DO, CYCLE, \
+   EXIT, and DO-loop constructs where appropriate. \
+4. 3-way arithmetic IFs converted to IF .. THEN .. ELSE constructs. \
+5. TYPE and INTENT declarations inserted for all subroutine and \
+   function arguments. \
+6. Computed GOTOs converted to SELECT CASE constructs. \
+7. Embedded blanks are removed from numerical constants. They are illegal \
+   in free format.
```

Comment:
MacPorts will run all of these lines together, displaying all of it as a single paragraph, which won't look good. Insert explicit newlines using `\n`. See the example in the long_description section of https://guide.macports.org/#reference.keywords

Reply:
OK